soundbay/conf/preprocessors/_preprocessors_unit.yaml

```diff
@@ -2,8 +2,8 @@
 _preprocessors:
   spectrogram:
     _target_: torchaudio.transforms.Spectrogram
-    n_fft: 1024
-    hop_length: 256
+    n_fft: ${data.n_fft}
+    hop_length: ${data.hop_length}
```

Comment:
did you change the conf files in the data folder to include these parameters? So it won't break experiments who use them

Reply:
The normal preprocessor.yaml file already has that, just updating preprocessing_unit.
Also datafiles do have these parameters.